RealestateApi/Validators/AddCustomerValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using RealEstateApi.Dto.Request;
+
+
+namespace RealEstateApi.Validators
+{
+    public class AddCustomerValidator : AbstractValidator<CustomerRequestDto>
+    {
+        public AddCustomerValidator()
+        {
+            RuleFor(x => x.Name).Length(5, 100).NotEmpty().WithMessage("Name is required.");
+            RuleFor(x => x.Email).Length(8, 150).EmailAddress().NotEmpty().WithMessage("A valid email is required");
```

Comment:
It makes sense to check if it's empty first, and if it's not empty  to check length and then if it's an email address, because if the property is null or empty it doesn't make sense to do the other validations